cmd/init_mon.go

```diff
@@ -66,7 +66,13 @@ func bootstrapMon() {
 	}
 
 	// Read ENV and search for a value for rgwPort
-	rgwPortEnv := os.Getenv("RGW_CIVETWEB_PORT")
+	rgwPortEnv := os.Getenv("RGW_FRONTEND_PORT")
+	if len(rgwPortEnv) > 0 {
```

Comment:
```suggestion
	if rgwPortEnv != "" {
```

This is more common.